data/vocab/lite.ttl

```diff
@@ -576,6 +603,12 @@ bld:subOrganization a rdf:Property ;
     rdfs:domain bld:Organization ;
     rdfs:range bld:Resource .
 
+bld:researchOrganizationRegistry a rdf:Property ;
+	rdfs:label "research organization registry"@en ;
+	rdfs:comment "The link to the organization's research organization registry page. "@en ;
+    rdfs:domain bld:Organization ;
+    rdfs:subPropertyOf bld:link .
```

Comment:
Should follow this example

scholar:orcid a rdf:Property ;
    rdfs:label "ORCID"@en ;
    rdfs:comment "An ORCID persistent digital identifier for a researcher."@en ;
    rdfs:domain bf:Person ;
	rdfs:range link:ORCID  .

Reply:
Need to add the ROR class as well. This will be referenced in the ROR property